Source/SvgNamespace.cs

```diff
@@ -1,9 +0,0 @@
-﻿
-
-namespace Svg
-{
-    public static class SvgNamespace
-    {
-        public const string UriString = "http://www.w3.org/2000/svg";
```

Comment:
Removing this changes the public API. I have no idea if this is used somewhere, but if it is, it will break the code. I'm a bit reluctant to introduce breaking changes for a dot release. 
On the other hand, the XXE mitigation also changes the behavior and will break existing applications, so maybe we should at least change the minor version for the next release. Strictly speaking, if following semantic versioning, each such change would mean a new major version, but I'm not sure about that... @tebjan - what do you think?

Regardless, any change in the public API also needs to be described in the release notes under `Changes`.

Reply:
In PR #760, public API `SvgAttributeAttribute.SvgNamespace` was removed when `SvgNamespace` class was created.
It's possible to leave `SvgNamespace` class, but I've removed it because having this class makes code dirty.

Reply:
Thanks, you are right - I wasn't aware that this has been added only after the last release!